azure-pipelines-official.yml

```diff
@@ -2,12 +2,14 @@ resources:
 - repo: self
   clean: true
 
-variables:
-  SignType: real
-  SkipTests: false
-  SkipApplyOptimizationData: false
-  IbcSourceBranchName: 'default'
-  IbcDropId: 'default'
```

Comment:
Do we have a link to the issue tracknig giving us an equivalent feature?

Reply:
PR with feature spec is in the comment: https://github.com/Microsoft/azure-pipelines-yaml/pull/129